deny.toml

```diff
@@ -62,10 +72,6 @@ highlight = "all"
 workspace-default-features = "allow"
 external-default-features = "allow"
 allow = []
-deny = [
-    { crate = "openssl", use-instead = "rustls" },
-    { crate = "openssl-sys", use-instead = "rustls" },
-]
```

Comment:
This is concerning. I am guessing that random alloy version I mentioned is pulling openssl in?

Reply:
Yeah I mean we had issues with OpenSSL in the past and I don't see how anything has changed in this regard. But I am fairly sure we can just get rid of `rig`, no? It's only used in one place AFAIT

Reply:
Yes, I got rid of `rig` and just copied in the [function](https://github.com/matter-labs/anvil-zksync/blob/90b81c0c4c6cf007aac0dccabdc22b9b41f2c0f5/crates/core/src/node/zksync_os.rs#L896) we were using from there. 👍